integration/integration_test.go

```diff
@@ -1281,25 +1281,26 @@ func testEscapeSequenceNoTrigger(t *testing.T, terminal *Terminal, sess <-chan e
 
 type localAddr struct {
```

Comment:
Should we rename the type to `localAddrs`? Should we instead be using multiple `localAddr` instances, or keeping a map of them elsewhere?

The fact that it holds multiple addresses is not something I'd expect from the present type name.

```suggestion
type localAddrs struct {
```